server/src/main/java/org/opensearch/action/ActionModule.java

```diff
@@ -449,6 +449,7 @@ public class ActionModule extends AbstractModule {
     private final SettingsFilter settingsFilter;
     private final List<ActionPlugin> actionPlugins;
     private final Map<String, ActionHandler<?, ?>> actions;
+    private ActionRegistry actionRegistryMap;
```

Comment:
Everything around this is `final`. Should this also be?

Reply:
Thats a good point. I was debating but it can be a final because the instance would not change. 
Will update.